UserTools/DirectionGridCheck/DirectionGridCheck.cpp

```diff
@@ -0,0 +1,245 @@
+#include "DirectionGridCheck.h"
+
+DirectionGridCheck::DirectionGridCheck():Tool(){}
+
+
+bool DirectionGridCheck::Initialise(std::string configfile, DataModel &data){
+
+  /////////////////// Useful header ///////////////////////
+  if(configfile!="") m_variables.Initialise(configfile); // loading config file
+  //m_variables.Print();
+
+  m_variables.Get("OutputFile", OutputFile);
+  m_variables.Get("ShowEvent", showEvent);
+  m_variables.Get("verbosity", verbosity);
+
+  fOutput_tfile = new TFile("DGCheck.root", "recreate");
+  AngularPlot = new TGraph();
```

Comment:
these TGraphs are not deleted, and TGraphs by default do not become owned by the current file